From 00d836686675f6495dfb91564f75c60c9e88edea Mon Sep 17 00:00:00 2001
From: Florian Schmaus <flo@geekplace.eu>
Date: Sun, 5 Dec 2021 12:48:42 +0100
Subject: [PATCH] cmake: add DOCDIR variable

Some distributions, e.g. Gentoo, require a particular naming scheme
for /usr/share/doc. In case of Gentoo, the package version is also
included, so it would be for example /usr/share/doc/etlegacy-2.78.1.
---
 cmake/ETLInstall.cmake | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/cmake/ETLInstall.cmake b/cmake/ETLInstall.cmake
index c0455aa9e111..5ecea9e67e59 100644
--- a/cmake/ETLInstall.cmake
+++ b/cmake/ETLInstall.cmake
@@ -2,6 +2,10 @@
 # Install
 #-----------------------------------------------------------------
 
+if(NOT DOCDIR)
+	set(DOCDIR "${INSTALL_DEFAULT_SHAREDIR}/doc/etlegacy")
+endif()
+
 # description file - see FS_GetModList
 install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/misc/description.txt"
 	DESTINATION "${INSTALL_DEFAULT_MODDIR}/${MODNAME}"
@@ -45,7 +49,7 @@ if(UNIX AND NOT APPLE)
 		DESTINATION "${INSTALL_DEFAULT_SHAREDIR}/man/man6"
 	)
 	install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/docs/INSTALL.txt"
-		DESTINATION "${INSTALL_DEFAULT_SHAREDIR}/doc/etlegacy"
+		DESTINATION "${DOCDIR}"
 	)
 	install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/COPYING.txt"
 		DESTINATION "${INSTALL_DEFAULT_SHAREDIR}/licenses/etlegacy"
-- 
2.32.0

